core/blocktree/blocktree.go

```diff
@@ -18,12 +18,12 @@ package blocktree
 
 import (
 	"fmt"
+	"github.com/ChainSafe/gossamer/core/types"
```

Comment:
This isn't properly sorted. Shouldn't CI report this?

Reply:
Yes it should have...I will look into that